fulfillment-feature-config-example.js

```diff
@@ -17,6 +17,9 @@ window.INSTORE_CONFIG = {
       reverseOrder: { // Enable order reserval (default false)
         enabled: true,
       },
+      allowStoresToCancelOrder: { // Changes the refuse order button to cancel order
```

Comment:
I think we could insert the information ```default false```
```suggestion
      allowStoresToCancelOrder: { // Changes the refuse order button to cancel order (default false)
```